daemon/main.go

```diff
@@ -833,7 +833,11 @@ func runDaemon() {
 
 	server.ConfigureAPI()
 
-	d.SendNotification(monitor.AgentNotifyStart, time.Now().String())
+	repr, err := monitor.TimeRepr(time.Now())
+	if err != nil {
+		log.WithError(err).Warn("Failed to notify monitor about agent start")
+	}
+	d.SendNotification(monitor.AgentNotifyStart, repr)
```

Comment:
Why do you send a notification when `err != nil` and you've just logged above that you're not sending a notification?
Put this into an `else` clause.